renku/core/management/migrations/m_0009__new_metadata_storage.py

```diff
@@ -226,6 +232,9 @@ def extract_run_uuid(run_id: str) -> str:
 
     plan_id = Plan.generate_id(uuid=uuid)
 
+    def get_mime_type(entity: Union[Entity, Collection]) -> List[str]:
+        return [DIRECTORY_MIME_TYPE] if isinstance(entity, Collection) else ["application/octet-stream"]
```

Comment:
Should be `renku.core.management.migrations.models.v9.Collection` and `renku.core.management.migrations.models.v9.Entity` here. `Collection`/`Entity` as imported at the top are the new ones from `renku.core.models.entity`